MULTI.TRA to chinese

Lzw104522773 opened the pull request:
MULTI.TRA to chinese

Lzw104522773 committed:
MULTI.TRA to chinese

MULTI.TRA to chinese

MephistoSatanDevil committed:
fix the gibberish problem

MephistoSatanDevil committed:
fix the gibberish problem

MephistoSatanDevil committed:
fix the gibberish problem

Lzw104522773 committed:
Merge pull request #1 from MephistoSatanDevil/main

fix the gibberish problem

skellytz commented:
Last time I tested the `WEIDU_OS` variable for setup prompts, Project Infinity couldn't properly understand it. I don't think that issue has been solved. Seeing as _setup.tra_ is only for WeiDU prompts and is excluded from character conversion, simply make it UTF-8 and don't split it into OS-specific files. For now we'll just have to assume Chinese players use Console or Terminal with UTF-8 support.

Lzw104522773 committed:
update Simplified Chinese translation

I have made modifications based on your email suggestions, and now using the V249 version of Weidu to select Chinese installation will no longer have garbled characters.

skellytz commented on arestorationp/arestorationp.tp2:
```diff
@@ -94,9 +94,8 @@ LANGUAGE ~Foeroyskr (BG2EE) (by inscrutable horse)~
 LANGUAGE ~Simplified Chinese (by Sebastian C)~
 		 ~schinese~
          ~arestorationp/language/english/game.tra~
-         ~arestorationp/language/english/setup.tra~
-         ~arestorationp/language/schinese/game.tra~
```
Keep this. US-ASCII is part of UTF-8, so if any new untranslated strings are added, they'll work for Chinese players, too.

Lzw104522773 committed:
Update arestorationp.tp2

Lzw104522773 commented:
OK！

skellytz commented:
Thanks!
